tasks/test.yml

```diff
@@ -17,13 +17,37 @@ vars:
 tasks:
   static:
     desc: Runs all static tests
-    deps: [lint, phpstan, phpunit:static, phpcs]
     cmds:
+      - |
+        set +e
+        task test:lint
+        STATUS_LINT=$?
+        task test:phpstan
+        STATUS_PHPSTAN=$?
+        task test:phpunit:static
+        STATUS_PHPUNITSTATIC=$?
+        task test:phpcs
+        STATUS_PHPCS=$?
+
+        if [ $STATUS_LINT -ne 0 ] || [ $STATUS_PHPSTAN -ne 0 ] || [ $STATUS_PHPUNITSTATIC -ne 0 ] || [ $STATUS_PHPCS -ne 0 ]; then
+          exit 1
+        fi
```

Comment:
🤔 this means these are all now run sequentially, and not in parallel. On the one hand, these are fast, on the other, parallel is even faster?

_however_, given this is an issue or feature request in Task itself, I'd love for us to instead fix or improve this upstream, as was linked in the original issue? See https://github.com/go-task/task/issues/1246